Equilobe.DailyReport.Models/BitBucket/Markup.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Equilobe.DailyReport.Models.BitBucket
+{
+    public enum Markup
+    {
+        markdown,
```

Comment:
Capital first letter?

Reply:
this is the exact string that is received from the request